python/ray/experimental/serve/global_state.py

```diff
@@ -0,0 +1,80 @@
+import time
+from collections import defaultdict, deque
+
+import ray
+from ray.experimental.serve.kv_store_service import KVStoreProxyActor
+from ray.experimental.serve.queues import CentralizedQueuesActor
+from ray.experimental.serve.utils import logger
+from ray.experimental.serve.server import HTTPActor
+
+# TODO(simon): this will be moved in namespaced kv stores
+
+
+class GlobalState:
+    """Encapsulate the global state in the serving system.
+
+    Warning:
+        Currently the state resides inside driver process. The state will be
+        moved into a key value stored service AND a supervisor service.
+    """
+
+    def __init__(self):
+        #: holds all actor handles.
```

Comment:
What is the `#:` syntax? Please make the comments standard everywhere (i.e., capitalize first letter, end with period.

Reply:
`#:` syntax is the sphinx class/object variable documentation notation. https://stackoverflow.com/questions/8649105/how-to-show-instance-attributes-in-sphinx-doc